internal/mocks/mock_token_serializer.go

```diff
@@ -0,0 +1,70 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: token_serializer.go
+//
+// Generated by this command:
+//
+//	mockgen -source token_serializer.go -destination ../../internal/mocks/mock_token_serializer.go -package mocks OpenFGADatastore
```

Comment:
@cikasfm did you run this command by hand?

PLease add this line to the `token_serializer.go` file to generate this on every build: `//go:generate mockgen -source token_serializer.go -destination ../../internal/mocks/mock_token_serializer.go -package mocks`